### ISSUE TITLE
A11y: Consider writing a VPAT for the Geoportal

### ISSUE THREAD

**Issue by ewlarson** [geobtaa/geoportal#600]
Many SaaS vendors (like [Gimlet](https://gimlet.us/files/Gimlet_VPAT2.4INT_2023.pdf)) include a [VPAT](https://en.wikipedia.org/wiki/Voluntary_Product_Accessibility_Template) accessibility report alongside their product.

A VPAT is typically used to document accessibility for buyers/purchasers of technology, but we could also use the VPAT as a formal document of the accessibility goals and achievements of the BTAA Geoportal. 

VPAT templates:
https://www.itic.org/policy/accessibility/vpat

It's less common in Open Source Software, but organizations like ArchivesSpace have adopted VPATs alongside their major releases: https://archivesspace.org/application/accessibility-vpat

Something for the GeoBlacklight community to consider, too.
